docs/ref/contrib/gis/install/spatialite.txt

```diff
@@ -21,14 +21,14 @@ In any case, you should always be able to :ref:`install from source
 __ https://www.gaia-gis.it/fossil/libspatialite
 __ https://www.gaia-gis.it/gaia-sins/
 
-.. _spatialite_source:
-
 .. admonition:: ``SPATIALITE_LIBRARY_PATH`` setting required for SpatiaLite 4.2+
 
-    If you're using SpatiaLite 4.2+, you must put this in your settings::
+    If you're using SpatiaLite 4.2+, you must put this in your **Django** ``settings.py``::
```

Comment:
Did you confuse it with some other settings? I ask because "in your settings" is used a whole lot in the documentation.